src/nacl/bindings/crypto_pwhash.py

```diff
@@ -267,10 +281,41 @@ def crypto_pwhash_scryptsalsa208sha256_str_verify(passwd_hash, passwd):
     return True
 
 
-def crypto_pwhash_argon2i(outlen, passwd, salt, opslimit, memlimit, alg):
+def _check_argon2_limits_alg(opslimit, memlimit, alg):
+
+    if (alg == crypto_pwhash_ALG_ARGON2I13):
+        if memlimit < crypto_pwhash_argon2i_MEMLIMIT_MIN:
+            raise exc.ValueError('memlimit must be at least {0} bytes'.format(
+                                 crypto_pwhash_argon2i_MEMLIMIT_MIN))
+        elif memlimit > crypto_pwhash_argon2i_MEMLIMIT_MAX:
+            raise exc.ValueError('memlimit must be at most {0} bytes'.format(
+                                 crypto_pwhash_argon2i_MEMLIMIT_MAX))
+        if opslimit < crypto_pwhash_argon2i_OPSLIMIT_MIN:
+            raise exc.ValueError('opslimit must be at least {0}'.format(
+                crypto_pwhash_argon2i_OPSLIMIT_MIN))
+        elif opslimit > crypto_pwhash_argon2i_OPSLIMIT_MAX:
+            raise exc.ValueError('opslimit must be at most {0}'.format(
+                crypto_pwhash_argon2i_OPSLIMIT_MAX))
+
+    elif (alg == crypto_pwhash_ALG_ARGON2ID13):  # pragma: no branch
```

Comment:
For testing purposes in `cryptography` we typically make this an else with an `assert alg == crypto_pwhash_ALG_ARGON2ID13`. That way the branch is covered but we make an assertion in case a different `alg` is passed.